homeassistant/components/speedtestdotnet/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "SpeedTest.net",
```

Comment:
This title does not need translating, and can thus be removed (the one from the manifest will be used instead).
```suggestion
```